tests/run_fastseq_tests.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
-
+rm -rf build/
+rm ngram_repeat_block_cuda*.so
+pip install --editable .
 for fastseq_py_test_file in $(find tests/ -name "test_*.py")
 do 
   if [[ $fastseq_py_test_file == *"test_fairseq_optimizer"* ]]; then
       echo "Running $fastseq_py_test_file with EL Attention"
-      USE_EL_ATTN=1 python $fastseq_py_test_file
+      PYTHONIOENCODING=utf8 USE_EL_ATTN=1 python3 $fastseq_py_test_file
```

Comment:
just curious what's `PYTHONIOENCODING` for?

Reply:
with the transformers docker, it defaults to ascii which causes the tests to fail on non ascii input